google-map-search.html

```diff
@@ -147,13 +169,18 @@
      * Perform a search using for `query` for the search term.
      */
     search: function() {
-      if (this.query && this.map) {
+      if (this.map && (this.query || this.type)) {
         var places = new google.maps.places.PlacesService(this.map);
 
         if (this.types && typeof this.types == 'string') {
           var types = this.types.split(' ');
         }
 
+        if (this.type && typeof this.type == 'string') {
+          var typeArray = this.type.split(' ');
+          var type = typeArray[0];
```

Comment:
You're right that the precedence has already been set :) I guess `places.textSearch` is cool with undefined values for keys?